website/docs/docs/deploy/ci-jobs.md

```diff
@@ -63,12 +63,13 @@ If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cl
 
 
 1. Set up a CI job with the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint using `"job_type": ci` or from the [dbt Cloud UI](#set-up-ci-jobs).
```

Comment:
I would add a note about turning the trigger to run on PRs off (or is it automatically toggled off for non-native integration setups?

Reply:
thanks for flagging, @kbrock91 !

yep, it's automatically turned off and includes a helper message too: 

![image](https://github.com/dbt-labs/docs.getdbt.com/assets/107218380/cf93d2e8-201b-44c4-af33-b87612d2a319)